### DISCUSSION TITLE
Netdata installation ABORTED - Static binary checksum validation failed.

### DISCUSSION THREAD

**Discussion by xujh0225** [netdata/netdata#14011] [Q&A] [answered]
I have been trying to install offline Netdata on one of Redhat Linux 6.9 server , but it keeps failing with following error:

ABORTED Static binary checksum validation failed. Usually this is a result of an older copy of the file being cached somewhere upstream and can be resolved by retrying in an hour.

**Comment by MrZammler** [comment, marked as answer]
Hi @xujh0225 

I assume this is a repeatable problem, right? A couple of questions if you don't mind, where are you located, and can you please provide the whole procedure you're following to install?

**Reply by xujh0225** [reply]
1. download:  wget -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh && sh /tmp/netdata-kickstart.sh --prepare-offline-install-source ./netdata-offline
2. install : 
![image](https://user-images.githubusercontent.com/18331663/202761434-258dc76e-978e-4ca0-b8ba-4afd5e978e98.png)
![image](https://user-images.githubusercontent.com/18331663/202761745-1340da9d-3f9a-4888-a29b-fa4353592660.png)

**Reply by xujh0225** [reply]
In addition, Some can be installed normally, part of others failed . Do you have a idea to check it ? Thanks.  
The servers version contains redhat 6.7 / 6.9 and CentOS release 6.9 (Final).
